bcipy/acquisition/tests/test_client.py

```diff
@@ -54,6 +54,44 @@ def __init__(self, *args, **kwargs):
         self.mock_channels = ['ch' + str(i) for i in range(num_channels)]
         self.mock_data = list(mock_data(num_records, num_channels))
 
+    def test_acquisition_exceptions(self):
```

Comment:
I'd name this test to assert the exception you're trying to throw. Is it unable to connect to a device or was a message misunderstood? Only test one at a time!

Reply:
Gotcha, I specified so that it tests only for NoneType device (cannot connect)